packages/react-composites/src/composites/ChatComposite/Strings.tsx

```diff
@@ -13,4 +13,19 @@ export interface ChatCompositeStrings {
    * Chat list header text
    */
   chatListHeader: string;
+  /* @conditional-compile-remove(file-sharing) */
+  /**
+   * Arialabel strings for file sharing upload and download file cards
+   */
+  fileSharing: {
```

Comment:
I recommend keeping this structure flat.
Nesting these strings under `fileSharing` is verbose.
The name `uploadFileButton` suggests that it is a file sharing related string.
Similarly renaming `removeFile` to `removeFileUpload` eliminates the need for nesting this string in `fileUploadCards` attribute.
Moreover, by exposing `fileUploadCards` attribute, we are leeking the underlying implementation details from composite which is not advised.

Reply:
done